Database/src/main/resources/changelog1.xml

```diff
@@ -811,4 +811,31 @@
         </insert>
     </changeSet>
 
+    <changeSet author="CAF" id="1520597520212-1">
+        <createTable tableName="tbl_global_config">
+            <column name="key" type="VARCHAR(255)">
+                <constraints nullable="false" primaryKey="true"/>
+            </column>
+            <column name="default" type="CLOB">
+                <constraints nullable="false"/>
+            </column>
+            <column name="description" type="CLOB"/>
```

Comment:
Let's make the description mandatory.

Reply:
Withdrawing this comment as forcing the user may just provide a dummy description if they are forced.